salt/_modules/metalk8s_service_configuration.py

```diff
@@ -35,13 +38,19 @@ def get_service_conf(
 
     .. code-block:: bash
 
-        salt-call metalk8s_service_configuration.get_service_conf "metalk8s-monitoring" "metalk8s-prometheus-config"
+        salt-call metalk8s_service_configuration.get_service_conf "metalk8s-monitoring" "metalk8s-prometheus-config" default_csc
     """
 
     if not configmap_name:
         raise Exception(
             'Expected a ConfigMap name but got {}'.format(configmap_name)
         )
+    if not default_csc:
+        raise Exception(
+            'Expected default CSC for ConfigMap {} but got {}'.format(
+                configmap_name, default_csc
+            )
+        )
```

Comment:
In a module, you should use `CommandExecutionError` rather than an `Exception`.
Also, for the second `if` I would check that the argument is a `dict` rather than checking if it's evaluated to false or not.
```suggestion
    if not configmap_name:
        raise CommandExecutionError(
            'Expected a ConfigMap name but got {}'.format(configmap_name)
        )
    if not isinstance(default_csc, dict):
        raise CommandExecutionError(
            'Expected default CSC for ConfigMap {} but got {}'.format(
                configmap_name, default_csc
            )
        )
```

Reply:
Done